src/Browser.ts

```diff
@@ -501,13 +495,13 @@ export class Browser {
 
         const res = await got<T>(this.#forceAbsoluteURL(url), options);
 
-        if (res.statusCode >= 400) {
+        if (res?.statusCode >= 400) {
```

Comment:
Can res ever by null here? I think `got` either returns the object, or throws an error. There is no inbetween.